gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -0,0 +1,169 @@
+import { Result, WriteResponse } from '@chainlink/gauntlet-core'
+import { TransactionResponse, SolanaCommand, RawTransaction } from '@chainlink/gauntlet-solana'
+
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+import { PublicKey, SYSVAR_RENT_PUBKEY, Keypair } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
+import { boolean } from '@chainlink/gauntlet-core/dist/lib/args'
+
+export const wrapCommand: any = (command) => {
```

Comment:
Please remove the `any` type. I saw it in more places, please remove them.

Reply:
Added to bypass some errors of this type:
`The inferred type of ... cannot be named without a reference to ...`, I will check again to see the issue there.

Reply:
Upgrading anchor dependency to 0.20 solved this